### PR TITLE
Add conversation for removing events

### PR DESCRIPTION
**Pull request by kwunkiuma**
The user can now remove events by interacting with the bot.

The bot prompts the user for an event ID and asks the user for confirmation before deleting the event.

### CODE REVIEW

File: DiscordBot/Commands/EventCommands.cs [hunk 1]
```diff
@@ -14,6 +14,7 @@ internal class EventCommands
         private static readonly string[] EventOperations =
         {
             "create",
+            "remove",
             "rm",
```
Comment (FridaTveit) [inline DiscordBot/Commands/EventCommands.cs:18]: What's `rm` for? 🙂

**Review by FridaTveit — COMMENTED** [review]
One comment, otherwise looks good 🙂

File: DiscordBot/Commands/EventCommands.cs (hunk 1 above)
Comment (kwunkiuma) [reply DiscordBot/Commands/EventCommands.cs:18]: It was the previous keyword before renaming the command to remove, but I thought I'd leave it in there as an option.

Should I remove this?

Comment (FridaTveit) [reply DiscordBot/Commands/EventCommands.cs:18]: I would definitely recommend removing it if it's not used! 🙂

**Review by FridaTveit — APPROVED** [review]
Looks good 🙂

**Pull request merged**